app/imports/components/Sprints/SingleSprint/index.js

```diff
@@ -4,19 +4,37 @@ import { withRouter } from 'react-router';
 
 import {
     Sprints,
+    actions as sprintActions,
 } from '/imports/api/sprints';
 
+import { Projects, isProjectModerator } from '/imports/api/projects';
+
 import SingleSprint from './SingleSprint.jsx';
 
+
 const composer = ({ params: { sprintId } }, onData) => {
     const sprintHandler = Meteor.subscribe('singleSprint', sprintId);
 
+
     if (sprintHandler.ready()) {
         const sprint = Sprints.findOne({ _id: sprintId });
 
-        onData(null, {
-            name: sprint.name,
-        });
+        const projectId = sprint.projectId;
+        const projectsHandler = Meteor.subscribe('singleProject', projectId);
+
+
+        if (projectsHandler.ready()) {
+            const project = Projects.findOne({ _id: projectId });
+            const userId = Meteor.userId();
+
+            onData(null, {
+                sprint,
+                toggleSprint: sprintActions.toggleSprint,
+                isProjectModerator,
```

Comment:
You can use this function here, in composer and send down only boolean prop - `canClose` or `isModerator`.